spec/std/system_spec.cr

```diff
@@ -18,4 +18,12 @@ describe System do
       cpu_count.should eq(shell_cpus)
     end
   end
+
+  describe "login" do
+    it "returns String when found" do
+      shell_login = `echo $USER`.strip
```

Comment:
All true, yet the returned value of $USER inside the docker runtime on Travis returns null.